### PR TITLE
12 chore add some notes on chapter 2 of working the command line

### PR DESCRIPTION
**Pull request by schalkneethling**
- chore: add some notes on chapter 2
- chore: notes for chapter two of wcl

### CODE REVIEW

**Comment by bpires** [comment]
The text looks perfectly fine to me, but I would like to maintain the same layout pattern of the "Chapter 1" notes. 

@schalkneethling, it would be ok if I add a commit to it after finishing my review to reorganize the content?

**Comment by schalkneethling** [comment]
> @schalkneethling, it would be ok if I add a commit to it after finishing my review to reorganize the content?

Please do! Thank you for asking.

File: working-the-command-line/chapter2/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+On macOS the most popular package manager is [`brew`](https://brew.sh/).
+
+## Python
+
+If you have chosen, or want to experiment, with Python to write command line utilities, ["Automating the boring stuff"](https://automatetheboringstuff.com/) is a great resource for this.
+
+### Related links
+
+- [gsudo - sudo for Windows](https://github.com/gerardog/gsudo)
+- [runas - Windows](<https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc771525(v=ws.10)?redirectedfrom=MSDN>)
```
Comment (schalkneethling) [inline working-the-command-line/chapter2/index.md:26]: @bpires, I am going to remove these two as they make more sense to have in chapter 4.

File: javascript-the-definitife-guide/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# JavaScript the definitife guide
```
Comment (bpires) [inline javascript-the-definitife-guide/README.md:1]:
```suggestion
# JavaScript the definitive guide
```

**Review by bpires — APPROVED** [review]
LGTM. I've updated the structure for more clarity and to mantain a pattern between chapter notes.

**Pull request merged**